src/ol/source/ImageCanvas.js

```diff
@@ -32,7 +32,7 @@ import ImageSource from '../source/Image.js';
  * projection. The canvas returned by this function is cached by the source. If
  * the value returned by the function is later changed then
  * `changed` should be called on the source for the source to
- * invalidate the current cached image. See @link: {@link module:ol/Observable~Observable#changed}
+ * invalidate the current cached image. See {@link module:ol/Observable~Observable#changed}
```

Comment:
@wallw-bits got this and a few others in #8708.

Reply:
oups, I missed that. I've merged #8708.